src/Data/List/Relation/Binary/Sublist/Propositional/Properties.agda

```diff
@@ -31,28 +30,29 @@ open import Relation.Unary using (Pred)
 
 private
   variable
-    b ℓ : Level
-    B : Set b
+    a ℓ : Level
+    A B : Set a
```

Comment:
Elsewhere, eg. `Data.List.Relation.Binary.Sublist.Setoid.Properties`, the `variable` block is enriched with bindings for 'typical' `List A` arguments. Can the same be done here, to simplify the binding prefixes in eg. L43 below, and elsewhere?

Reply:
Yup, nice idea!